src/api/tests/test_available.py

```diff
@@ -69,8 +69,8 @@ def test_domain_available_capitalized(self):
         self.assertTrue(check_domain_available("igorville.gov"))
         # input is lowercased so GSA.GOV should also not be available
         self.assertFalse(check_domain_available("GSA.gov"))
```

Comment:
Hm I think what happened was that earlier we manually mocked a response for gsa.gov and GSA.gov separately. I can update the tests since hypothetically we would no longer need to set test cases for the same domain names that are capitalized differently